### PR TITLE
check: restore force healthy

### PR DESCRIPTION
**Pull request by tmgordeeva**
Restores a force healthy/unhealthy option previously in 2.0 code.

### CODE REVIEW

File: kit/check/check.go [hunk 1]
```diff
@@ -68,9 +74,22 @@ func (c *Check) CheckHealth(ctx context.Context) Response {
 		Status: StatusPass,
 		Checks: make(Responses, len(c.healthChecks)),
 	}
+	override := c.manualOverride.Load().(bool)
+	if override {
+		if c.manualHealthState.Load().(bool) {
+			response.Status = "pass"
```
Comment (stuartcarnie) [inline kit/check/check.go]: nit: use `StatusPass` constant

File: kit/check/check.go [hunk 2]
```diff
@@ -68,9 +74,22 @@ func (c *Check) CheckHealth(ctx context.Context) Response {
...
+		} else {
+			response.Status = "fail"
```
Comment (stuartcarnie) [inline kit/check/check.go]: nit: use `StatusFail` constant

File: kit/check/check.go [hunk 3]
```diff
@@ -68,9 +74,22 @@ func (c *Check) CheckHealth(ctx context.Context) Response {
...
+		overrideResponse := Response{
+			Name:    "manual-override",
+			Message: "health manually overriden",
+		}
+		response.Checks = append(response.Checks, overrideResponse)
+	}
 	for i, ch := range c.healthChecks {
 		resp := ch.Check(ctx)
-		if resp.Status != StatusPass {
+		if (resp.Status != StatusPass) && !override {
```
Comment (jacobmarble) [inline kit/check/check.go]: nit: parens not needed

**Comment by jacobmarble** [comment]
Thanks @tmgordeeva !

**Pull request merged**